Marker/Marker/Parser/Token.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Token.swift
+//  Marker
+//
+//  Created by Htin Linn on 10/25/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
```

Comment:
something minor I came across very recently actually, is in a file like this you're not actually using Foundation for anything (like NSObject, so this import is useless) you could actually remove it if you wanted to. I don't think there's any benefit other than conciseness though

Reply:
Yeah. I try remove it when I remember but, most of the time, I don't.